src/signals/incident-management/containers/AreaContainer/AreaContainer.tsx

```diff
@@ -49,9 +58,19 @@ export const AreaContainer: FunctionComponent = () => {
 
   if (!area?.features || !incident) return null
 
+  const sidebar =
+    selection && selectedIncident ? (
+      <IncidentDetail
```

Comment:
There's no feedback to the user that data is being fetched. Probably better to show a spinner in this component (check with UX).